installer/darwin/create_dmg.sh

```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/env bash
+
+# Create a read-only disk image of the contents of a folder
+# This script is derived from the create-dmg utility
+# https://github.com/create-dmg/create-dmg
+
+# Bail out on any unhandled errors
+set -ex;
+
+VOLUME_NAME=$1
+APP_BUNDLE_PATH=$2
+DMG_PATH=$3
+
+VOLUME_ICON_FILE=installer/darwin/invest.icns
+BACKGROUND_FILE=installer/darwin/background.png
+EULA=LICENSE.txt
+
+SRC_FOLDER=$(cd $APP_BUNDLE_PATH > /dev/null; pwd)
+DMG_DIRNAME=$(dirname $DMG_PATH)
+DMG_DIR=$(cd $DMG_DIRNAME > /dev/null; pwd)
+DMG_NAME=$(basename $DMG_PATH)
+DMG_TEMP_NAME=$DMG_DIR/tmp.$DMG_NAME
+
+# remove .DS_Store from source folder, if there is one
+rm $SRC_FOLDER/.DS_Store || true
```

Comment:
Ooo I like the `|| true`, that's a nice way to handle the case where the folder doesn't exist.